mix.exs

```diff
@@ -52,7 +52,8 @@ defmodule EpochElixir.Mixfile do
       {:distillery, "~> 1.5.2", runtime: false},
       {:aewallet, github: "aeternity/elixir-wallet"},
       {:msgpax, "~> 2.1.1"},
-      {:bip0173, "~> 0.1.2"}
+      {:bip0173, "~> 0.1.2"},
+      {:erl_base58, "~> 0.0.1"}
```

Comment:
does this library actually do base58check?

Reply:
It encodes/decodes binaries/integers to base58 and vice versa

Reply:
as requested in #256, we need `base58c`

Reply:
does this library actually do base58check?

Reply:
No, we do it manually in the code

Reply:
we have the `quanterall/base58check`, why not use it?

Reply:
Because the prefix there is fixed to 4 bytes, we use prefix with 3 in our case

Reply:
`quanterall/base58check` can be used without prefix as well

Reply:
Yes, we can pass it like encoded = Base58Check.encode58check("", value_to_encode) and it will work, but when we decode -> Base58Check.decode58check(encoded) we get a return value:

Reply:
okay, lets keep it like this...

Reply:
yes, lets keep it, as is